src/components/page/body.vue

```diff
@@ -22,11 +20,26 @@ export default {
   name: 'PageBody'
 };
 </script>
+<script setup>
+import { computed, inject } from 'vue';
+
+const { router } = inject('container');
+const htmlClass = computed(() => ({
+  // `router` may be `null` in testing.
+  'full-width': router != null && router.currentRoute.value.meta.fullWidth
+}));
+</script>
 
 <style lang="scss">
-@import '../../assets/scss/variables';
+@import '../../assets/scss/mixins';
 
 #page-body {
+  @include clearfix;
```

Comment:
`.row` has clearfix styles. Now that `PageBody` won't use `.row`, I thought I'd add a clearfix to `PageBody`.